cli/src/diff_util.rs

```diff
@@ -1093,6 +1106,20 @@ pub fn show_types(
     })
 }
 
+pub fn show_names(
```

Comment:
nit: I'd prefer to put this after `diff_summary_char()` to leave that function closer to its caller